### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by sungam3r** [dotnet/docs#45378]
### Type of issue

Typo

### Description

This is the option to use if you feel that restating the accessibility when it matches what the language chooses by default is redundant and should be disalloed.

->

This is the option to use if you feel that restating the accessibility when it matches what the language chooses by default is redundant and should be disallowed.

disalloed -> disallowed

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%209

### Content source URL

https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler Breaking Changes - DotNet 9.md

### Document Version Independent Id

9a2ec6b8-6414-4aa7-4497-91a45d7be45a

### Platform Id

f9eb2f4e-cca5-aaed-0a78-e28b9a2ec4cf

### Article author

@BillWagner

### Metadata

* ID: 2022c144-1120-4c7d-a6d1-287aa056c169
* PlatformId: f9eb2f4e-cca5-aaed-0a78-e28b9a2ec4cf 
* Service: **dotnet-csharp**
* Sub-service: **whats-new**

[Related Issues](https://github.com/dotnet/docs/issues?q=is%3Aissue+is%3Aopen+9a2ec6b8-6414-4aa7-4497-91a45d7be45a)

**Comment by BillWagner** [comment]
Hi @sungam3r 

Thanks for reporting this.  I've opened dotnet/roslyn#77678 to address this.

When you spot these issues in the future, you can create a PR yourself if you'd prefer. Just click the "edit" button at the top of any file (it looks like a pencil), and that opens the source file in the GitHub editor for you to make changes.

Thanks again.

**Comment by vartaks** [comment]
@sungam3r @BillWagner 
I see on the relevant page https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%209 that the typo is already fixed.
Please close this issue.

**Comment by BillWagner** [comment]
Fixed in dotnet/roslyn#77678
